### ISSUE TITLE
Namespace does not match folder structure "\" Prefix?

### ISSUE THREAD

**Issue by wilwade** [researchgate/phpnsc#10]
Trying out the master branch and ran into a small issue.

> Namespace does not match folder structure, got campaign\components expected \campaign\components

It is not the root config setting. It could be that I don't have a vendor namespace at all on this project, but I think it should perhaps still be considered valid.

Any ideas I missed?

**Comment by bashofmann** [comment]
It's probably just a small thing but without your project this is hard to figure out. So it would be great if you could provide either a small project where this issue is reproducible or a test case for this issue.

Also feel free to send a pull request for this :)

**Comment by deividaspetraitis** [comment]
Same here for me. Config placed in the root of the project. Project is running on Laravel framework and got:

> <file name="/home/vagrant/Code/api.xxx.app/public_html/Modules/..">
> <error line="1" column="1" severity="error" message="Namespace does not match folder structure, got Modules\xxx \Modules\xxx" source="rg.tools.phpnsc" />
> </file>

```
{
  "vendor" : "",
  "folders" : {
    "root"    : "./",
    "include" : ["Modules"],
    "exclude" : ["vendor"]
  },
  "filetypes" : {
    "include" : [".php"],
    "exclude" : [".config.php"]
  },
  "output" : [{
    "class": "rg\\tools\\phpnsc\\CheckstyleOutput",
    "parameter": "storage/logs/phpnsc.xml"
  },{
    "class": "rg\\tools\\phpnsc\\ConsoleOutput",
    "parameter": ""
  }]
}
```

**Comment by danez** [comment]
This is because the vendor is empty.

Fixed in 3.0.3 (2160ebe1ae4114e4c56553e8aa8789ae822e238a)
